phat/tests-rs/src/lib.rs

```diff
@@ -275,7 +275,16 @@ mod tests {
                 .expect("Failed to upload core.js");
             offchain_rollup
                 .call_mut()
-                .config_core(code_hash, lens_api.clone())
+                .config_core(Core {
+                    driver: JsDriver::AsyncJsRuntime,
```

Comment:
Can we still keep a test for `JsDelegate` runtime?

Reply:
Added